includes/notes/class-wc-admin-notes-welcome-message.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * WooCommerce Admin: Welcome to WooCommerce Admin Note
+ *
+ * Adds welcome notes to store alerts area.
+ *
+ * @package WooCommerce Admin
+ */
+
+defined( 'ABSPATH' ) || exit;
+/**
+ * WC_Admin_Notes_Welcome_Message.
+ */
+class WC_Admin_Notes_Welcome_Message {
+	const NOTE_NAME = 'woocommerce-admin-welcome-note';
```

Comment:
Nitpicking here, but what do you think about renaming this to `wc-admin...` like in other notes.
```suggestion
	const NOTE_NAME = 'wc-admin-welcome-note';
```

Reply:
Yes, sure. updated it.